app/components/form/consumption-list.js

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+
+const ConsumptionListComponent = Component.extend({
```

Comment:
Could you write this component in the new octane style? Saves some migrations later

Reply:
Unfortunately this is not doable without updating a ton of other components too